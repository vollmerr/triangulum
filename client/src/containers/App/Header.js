import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <ul style={{ listStyleType: 'none', padding: 0 }}>
    <li>
      <Link to="/">home</Link>
    </li>
    <li>
      <Link to="/graph">graph</Link>
    </li>
    <li>
      <Link to="/history">history</Link>
    </li>
  </ul>
);

export default Header;
