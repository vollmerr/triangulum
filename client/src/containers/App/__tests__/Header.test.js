import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

let wrapper;
describe('App', () => {
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
