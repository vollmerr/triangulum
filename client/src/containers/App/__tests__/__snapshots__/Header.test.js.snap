// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<ul
  style={
    Object {
      "listStyleType": "none",
      "padding": 0,
    }
  }
>
  <li>
    <Link
      replace={false}
      to="/"
    >
      home
    </Link>
  </li>
  <li>
    <Link
      replace={false}
      to="/graph"
    >
      graph
    </Link>
  </li>
  <li>
    <Link
      replace={false}
      to="/history"
    >
      history
    </Link>
  </li>
</ul>
`;
